src/test/java/webserver/ControllerTest.java

```diff
@@ -57,4 +57,18 @@ public void testItReturns404PageWhenDetailNotFound() throws IOException {
 
     assertThat(htmlString, containsString("File Not Found"));
   }
+
+  @Test
+  public void testItReturnsResponseFromCustomDirectory() throws IOException {
+    String htmlString = controller.getDirectoryFile("/public/example.txt");
+
+    assertThat(htmlString, containsString("This is an example file"));
+  }
+
+  @Test
+  public void testItReturns404WhenFileFromCustomDirectoryNotFound() throws IOException {
```

Comment:
This is a good test for the current behavior, and I like that you have been descriptive in the name of the test.  Looking further at the name of the test, it suggests that maybe the controller should return more than just a String with the response body and maybe also include something about the status code.

In other words

```java
public Response getDirectoryFile(String path) ...
```

instead of `public String getDirectoryFile(...)`